org.eclipse.emfcloud.validation/src/main/java/org/eclipse/emfcloud/validation/EMFFacetConstraints.java

```diff
@@ -0,0 +1,197 @@
+/*******************************************************************************
+ * Copyright (c) 2019-2020 EclipseSource and others.
```

Comment:
I think it should be
```suggestion
 * Copyright (c) 2020 EclipseSource and others.
```